cloud_functions/flagCryptominer.js

```diff
@@ -0,0 +1,24 @@
+const { promisify } = require('util')
+const utils = require('./utils')
+
+
+const run = utils.withErrorReporting((mixpanel, message) => {
+  const userSubjectId = validate(message)
+  return promisify(mixpanel.people.set)(`google:${userSubjectId}`, { 'isCryptominer': true })
```

Comment:
Nice use of `promisify`. Note that this is ok to use because the mixpanel library doesn't rely on `this` internally, so wrapping the bare function works fine.

Reply:
Thanks for pointing that out. I didn't see any mention of that in the node docs. 😐  Can we be sure that the mixpanel library doesn't rely on `this`? And can we be sure that that will always be true? I'm wondering if it would be better to write it out longhand in this case.

Reply:
Okay. I'm less enamored of `promisify` now, but I'll leave it because risk seems low and I trust that failures will not be silent.